frontend/src/features/workflow-viewer/ui/GraphViewer.module.css

```
@@ -0,0 +1,45 @@
+.container {
+  width: 100%;
+  height: 100%;
+  min-height: 500px;
+  position: relative;
+  background: var(--bg-color);
+  border: 1px solid var(--glass-border);
+  border-radius: var(--radius-md);
+  overflow: hidden;
+}
+
+.container :global(.react-flow__edge-path) {
```

Stylelint (17.11.0)

[error] Unknown pseudo-class selector ":global" (selector-pseudo-class-no-unknown)

(selector-pseudo-class-no-unknown)